docs: API conclusion (#175)
RequestNetwork/requestNetwork: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md

---

> Request API is an abstraction over the Request Network for easy, but centralised, access. It's the fastest way to get started and build integrations.

Reviewer: ```suggestion
Request API is an abstraction over the Request Network for easy, but centralized, access. It's the fastest way to get started and integrate Request to your product.
```

## Pros
* Simple REST API, familiar for classic web developers
* No key management

> * Faster response time

Reviewer 2: With the buffer on the storage layer I'm not sure it's still the case. But I'm ok with leaving this here until we have measurements.

Author: read performance is still better.

> * Notifications on new request and received payments

Reviewer: ```suggestion
* Notifications on new requests and received payments
```

* Send requests to other API users based on their email, instead of their ETH address.
* Easy to build applications on top of Request (see [here](./3-api-apps))

> * Easy choice between fully public data or encrypted data on the network 

Reviewer: What do you mean ?
Do you mean it's easy to create encrypted requests ?

Author: yes, but the request will not be end-to-end encrypted

## Cons

> * centralised 

Reviewer 2: Nit: pros are all uppercase, cons are all lowercase :)

Author: ```suggestion
* Centralised 
```

> * centralised 

Reviewer: ```suggestion
* centralized 
```

America f**k yeah

> * centralised 

Reviewer: Well, the payment is not centralized and the data is still stored on a distributed ledger.

I would say `not fully decentralized`

> * users need a Request account to connect

Author: ```suggestion
* Users need a Request account to connect
```

> * encryption is not end-to-end (see [here](./4-api-encryption.md))

Author: ```suggestion
* Encryption is not end-to-end (see [here](./4-api-encryption.md))
```
